src/navigator.js

```diff
@@ -8,6 +9,21 @@ class Navigator {
         return reject(STATUS.ERROR_LOCATION);
       }
 
+      const cacheLocationMinutes = parseFloat(Storage.getItem(Storage.CACHE_LOCATION_MINUTES));
+      if (cacheLocationMinutes) {
+        try {
+          const lastLocation = JSON.parse(Storage.getItem(Storage.LAST_LOCATION));
```

Comment:
we should probably skip the rest if LAST_LOCATION is null, think it will error for the first call and go to the warn since there will be undefined fields

Reply:
👍